### DISCUSSION TITLE
Ability to translate the menu

### DISCUSSION THREAD

**Discussion by C0la149** [n3d1117/chatgpt-telegram-bot#193] [General]
Hello, is there any possibility that he would immediately give out such a message but in Russian?
![Telegram_oclSyeO1yo](https://user-images.githubusercontent.com/64633686/230686531-5b8f6efb-1e11-47ac-a671-9690da211b85.png)
And also that the buttons were in Russian too?
![vaWpRBpq8B](https://user-images.githubusercontent.com/64633686/230686572-35cd7557-b1ec-483c-8edb-8097ecfdea7f.png)
Many thanks in advance for your help :)

**Comment by Pumbaardian** [comment]
Привет!
Просто измени текст в файле telegram_bot.py
![image](https://user-images.githubusercontent.com/99145771/230722638-b344bb6f-94cd-458d-bf82-86ab2eb6c984.png)

**Reply by C0la149** [reply]
Понял принял, спасибо огромное тебе :)
